yarn-project/prover-client/src/orchestrator/orchestrator.ts

```diff
@@ -349,38 +332,34 @@ export class ProvingOrchestrator {
     setImmediate(safeJob);
   }
 
-  private getCircuitNameFromRequest(request: ProvingRequest): CircuitSimulationStats['circuitName'] | null {
-    switch (request.type) {
-      case ProvingRequestType.PUBLIC_VM:
-        return null;
-      case ProvingRequestType.PUBLIC_KERNEL_NON_TAIL:
-        switch (request.kernelType) {
-          case PublicKernelType.SETUP:
-            return 'public-kernel-setup';
-          case PublicKernelType.APP_LOGIC:
-            return 'public-kernel-app-logic';
-          case PublicKernelType.TEARDOWN:
-            return 'public-kernel-teardown';
-          default:
-            return null;
-        }
-      case ProvingRequestType.PUBLIC_KERNEL_TAIL:
-        switch (request.kernelType) {
-          case PublicKernelType.TAIL:
-            return 'public-kernel-tail';
-          default:
-            return null;
+  private emitCircuitSimulationStats(
+    circuitName: CircuitSimulationStats['circuitName'] | null,
+    inputSize: number,
+    outputSize: number,
+    duration: number,
+  ) {
```

Comment:
This should be moved to individual `CircuitProver` implementations as we'll want to separate stats for simulation vs simulation + proving